Source/Core/Core/HW/DSPLLE/DSPDebugInterface.cpp

```diff
@@ -257,9 +257,9 @@ void DSPDebugInterface::ToggleMemCheck(u32 address, bool read, bool write, bool
 // =======================================================
 // Separate the blocks with colors.
 // -------------
-int DSPDebugInterface::GetColor(u32 address)
+u32 DSPDebugInterface::GetColor(u32 address)
 {
-  static const int colors[6] = {
+  static const u32 colors[6] = {
```

Comment:
Same for this one. Although, this array is the same as back in `PPCDebugInterface::GetColor`...